### ISSUE TITLE
Unable to upload zip file with different extension

### ISSUE THREAD

**Issue by msoutopico** [pqina/filepond#411]
Hi there, 

### Summary
I'm trying to upload regular zip files with the `omt` extension, but they are rejected. 

However, if I change the extension to `zip`, then the file is accepted. 

My component looks like this: 
```
    <file-pond
      name="test"
      ref="pond"
      label-idle="Drop files here or <span class='filepond--label-action'>Browse</span>"
      allow-multiple="true"
      accepted-file-types="
                  zip, 
                  omt, 
                  application/omt, 
                  application/octet-stream, 
                  application/zip, 
                  application/x-zip, 
                  application/x-zip-compressed"
      v-bind:files="myFiles"
    />
```

### How to reproduce
Here's a sample file that gets uploaded 
https://www.dropbox.com/s/gahdvunawew4bpz/omegat.zip?dl=0
and the same file with different extension that is rejected:
https://www.dropbox.com/s/nhfmx2zg8vh3dy3/omegat.omt?dl=0

I have tested it here: https://codesandbox.io/s/vue-filepond-live-demo-7i4ck

### Expected and actual behaviour
Here's a screenshot: 
![image](https://user-images.githubusercontent.com/3659409/68954285-0198d880-07c4-11ea-8268-ff9dc6b41110.png)

I'd be grateful for any tips. Thanks.

**Comment by rikschennink** [comment]
You can use this tool to see how the browser sees the file: https://codepen.io/rikschennink/pen/NzRvbj

It’s possible it doesn’t know the extension and doesn’t assign a mime type. You can then use the the custom type validation hook (see docs).

**Comment by msoutopico** [comment]
Thanks Rik. 

When I upload the .omt file to the link you sent on Codepen, the `file.type` is empty. Since the extension is made up, I guess that makes sense. I've just grasped that the `accepted-file-types` attribute depends on what the browser sees.

I have found this section in the docs: https://pqina.nl/filepond/docs/patterns/plugins/file-validate-type/#custom-type-detection I suppose the custom type detection that I must code must rely on the `file.name` (well, more precisely, on the file extension)
